firefox_ui_harness/update.py

```diff
@@ -0,0 +1,4 @@
+from firefox_ui_harness import cli_update
+
+if __name__ == '__main__':
+    cli_update()
```

Comment:
I was thinking that we move the whole method from runtests.py to this file: https://github.com/mozilla/firefox-ui-tests/blob/master/firefox_ui_harness/runtests.py#L81. Maybe its better to also rename this file as `cli_update.py`.